pkg/kubectl/resource_filter.go

```diff
@@ -32,26 +30,7 @@ type FilterFunc func(runtime.Object, printers.PrintOptions) bool
 type Filters []FilterFunc
```

Comment:
There are no filters anymore, so this type should be removed.

Reply:
And how about the entire `resource_filter.go` file ?

Reply:
Since we only care about keeping the flag, I would be in favor of removing this file (and any references to it in factory_client_acces.go, get.go)